pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -15,6 +15,7 @@ import (
 	pkgrbac "github.com/rancher/rancher/pkg/rbac"
 	"github.com/rancher/rancher/pkg/user"
 	"github.com/sirupsen/logrus"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
`v1` and `metav1` refer to the same package. That should, IMHO, be simplified.

Reply:
Fixed